### PR TITLE
fix: enable quotations in input commands of guardian-admin.yml

### LINKED ISSUE #321: Quoted arugments on Guardian Admin workflows not being properly un/escaped.
### TL;DR

I tried running `apply -replace="random_id.github_abc_updater" -input=false -auto-approve` using a guardian admin workflow, but got 
```
  ╷
  │ Error: Invalid force-replace address "\"random_id.github_abc_updater\""
  │ 
  │ Must begin with a variable name.
  ╵
```

It worked correctly after removing the quotes.

According to hashicorp documentation, quotes should work. https://developer.hashicorp.com/terraform/cli/state/taint

### Expected behavior

_No response_

### Observed behavior

_No response_

### Environment Details

Team's infra repo.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by gjonathanhong**
Fixes #321.

### CODE REVIEW

**Comment by gjonathanhong** [comment]
Open to alternatives 👋

File: abc.templates/base-workflows/contents/guardian-admin.yml [hunk 1]
```diff
@@ -133,4 +133,4 @@ jobs:
         run: |-
           guardian run \
           -dir="${DIRECTORY}" \
-          -- ${TERRAFORM_COMMAND}
+          -- ${TERRAFORM_COMMAND//\"}
```
Comment (pdewilde) [inline abc.templates/base-workflows/contents/guardian-admin.yml:136]: What is going on here? This looks like you are adding a quote, not dropping.

Comment (gjonathanhong) [reply abc.templates/base-workflows/contents/guardian-admin.yml:136]: In bash, you can use pattern matching to substitute characters with the format `${parameter//pattern/string}`.

From `man bash`, 
> If there are two slashes separating _parameter_ and _pattern_, all matches of _pattern_ are replaced with _string_. If _string_ is null, matches of pattern are deleted and the ‘/’ following _pattern_ may be omitted.

So in this case, I'm matching all double quotes in the `TERRAFORM_COMMAND` string variable and removing it.

Comment (pdewilde) [reply abc.templates/base-workflows/contents/guardian-admin.yml:136]: TIL.

Might make sense to add a comment somewhere

**Review by pdewilde — APPROVED** [review]

File: abc.templates/base-workflows/contents/guardian-admin.yml (hunk 1 above)
Comment (sethvargo) [reply abc.templates/base-workflows/contents/guardian-admin.yml:136]: This is going to remove _all_ quotes, which could be unintended if my `TERRAFORM_COMMAND` is something like `terraform "foo bar"` where I intend "foo bar" to be a single argument. I think you want something like https://stackoverflow.com/questions/9733338/shell-script-remove-first-and-last-quote-from-a-variable

Comment (gjonathanhong) [reply abc.templates/base-workflows/contents/guardian-admin.yml]: Fair argument, but that also doesn't fix the initial problem.

It turns out, saving `inputs.command` into an environment variable `$TERRAFORM_COMMAND` messed with the way the arguments are parsed (I'm assuming it uses the default `IFS` delimiter to parse the variable).  
For example, an input of `plan "foo bar"` will parse to `["plan", "\"foo", "bar\""]`

But using the `inputs.command` directly in GitHub Actions expression syntax produced the desired result.

**Review by pdewilde — APPROVED** [review]

**Pull request merged**